src/Shouldly.Tests/InternalTests/DifferenceIndexConsolidatorTests.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using System.Collections.Generic;
-using NUnit.Framework;
 using Shouldly.DifferenceHighlighting;
+using Xunit.Extensions;
 
 namespace Shouldly.Tests.InternalTests
 {
-    [TestFixture]
     public class DifferenceIndexConsolidatorTests
     {
         const int MaxDiffLength = 5;
         const int MaxLengthOfStrings = 21;
 
-        [Test, TestCaseSource("IndexConsolidationTestCases")]
+        [Theory]
+        [PropertyData("IndexConsolidationTestCases")]
```

Comment:
Use `Xunit.MemberData` instead of obsolete `Xunit.Extensions.PropertyData` [more info](https://xunit.github.io/docs/test-migration.html)
The former is in Xunit v2, and the latter in Xunit v1. Also, there is no need to use `Xunit.Extensions` namespace in this situation.